docs/integrations/README.md

```diff
@@ -13,17 +13,21 @@ Airbyte uses a grading system for connectors to help users understand what to ex
 | Connector | Grade |
 |----|----|
 |[Amazon Seller Partner](./sources/amazon-seller-partner.md)| Alpha |
+|[Amplitude](./sources/amplitude.md)| Beta |
 |[Appstore](./sources/appstore.md)| Alpha |
 |[Asana](./sources/asana.md) | Beta |
+|[AWS CloudTrail](./sources/aws-cloudtrail.md)| Beta |
 |[Braintree](./sources/braintree.md)| Alpha |
 |[ClickHouse](./sources/clickhouse.md)| Beta |
+|[Db2](./sources/db2.md)| Beta |
 |[Drift](./sources/drift.md)| Beta |
 |[Exchange Rates API](./sources/exchangeratesapi.md)| Certified |
-|[Facebook Marketing](./sources/facebook-marketing.md)| Beta |
+|[Facebook Marketing](./sources/facebook-marketing.md)| Certified |
```

Comment:
I would keep as beta for a few weeks -- we're still working out kinks espeically with insights streams

Reply:
Done!